test/benchmark/common/entrypoint.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: MPL-2.0
+# Entrypoint for the benchmark VM
```

Comment:
I think `entrypoint` is a bit of vague. Calling it `bencher`, `run_bench`, `bench_runner`, or something similar would be more self-describing.